lib/main_publiccloud.pm

```diff
@@ -62,7 +62,7 @@ sub load_publiccloud_consoletests {
     loadtest 'console/journalctl';
     loadtest 'console/procps';
     loadtest 'console/suse_module_tools';
-    loadtest 'console/libgcrypt';
+    loadtest 'console/libgcrypt' if is_sle '12-SP5+';
```

Comment:
You are saying `The console/libgcrypt test module used to be in EXCLUDE_MODULES variable for SLE 12-SP4.`
But then you are doing the opposite here. Shouldn't it be `if is_sle '12-SP4-'`  or `if < is_sle '12-SP5'` ?

Reply:
No. 12-SP5+ means it will be EXLUDED for 12-SP4. As it was before.

Reply:
```suggestion
    loadtest 'console/libgcrypt' unless (is_sle '12-SP4');
```

Reply:
As you with lö Felix

Reply:
> EXLUDED

ofc (facepalm myself)